app/src/sharedTest/java/org/oppia/app/story/StoryFragmentTest.kt

```diff
@@ -80,6 +82,18 @@ class StoryFragmentTest {
     }
   }
 
+  @Test
+  fun testStoryFragment_changeConfiguration_textViewIsShownCorrectly() {
```

Comment:
This test case is not passing at the same time if I change the launcher activity to `StoryFragmentTest` and set storyId to `TEST_STORY_ID_1`, I do not see any item with `First Story` as title. 

Also, it would better to just have a test case similar to `testStoryFragment_correctStoryCountLoadedInHeader` with change of orientation in it.

something like this:

```
  @Test
  fun testStoryFragment_changeConfiguration_correctStoryCountLoadedInHeader() {
    launch<StoryFragmentTestActivity>(createTestActivityIntent(TEST_STORY_ID_1)).use {
      onView(isRoot()).perform(orientationLandscape())
      val headerString: String = getResources().getQuantityString(R.plurals.story_total_chapters, 3, 1, 3)
      onView(withId(R.id.story_chapter_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(0))
      onView(atPositionOnView(R.id.story_chapter_list, 0, R.id.story_progress_chapter_completed_text)).check(
        matches(
          withText(headerString)
        )
      )
    }
  }
```

Reply:
There was a typo. It was supposed to be Second Story. I have changed that. Also added the new test case.